addons/payment_payulatam/models/payment_acquirer.py

```diff
@@ -46,16 +46,27 @@ def _payulatam_generate_sign(self, values, incoming=True):
         :rtype: str
         """
         if incoming:
+            # "Confirmation" and "Response" pages have a different way to calculate what they call the `new_value`
+            if values.get('payulatam_is_confirmation_page'):
+                # https://developers.payulatam.com/latam/en/docs/integrations/webcheckout-integration/confirmation-page.html#signature-validation
+                # For confirmation page, PayU Latam round to the first digit if the second one is a zero
+                # to generate their signature.
+                # e.g:
+                #  150.00 -> 150.0
+                #  150.26 -> 150.26
+                # This happens to be Python 3's default behavior when casting to `float`.
+                new_value = float(values.get('TX_VALUE'))
```

Comment:
Probably overkill, but it might cover cases that I didn't thought about so it might be a good idea indeed :) 
Note that I did use `float_split_str` instead (and use `%s` instead of `%d`). The reason is that `float_split` itself call `float_split_str` and then convert into integer which is not something that we really need. So for "performance reason" in our case  `float_split_str` looks to be more appropriate

Reply:
I'm afraid this doesn't *exactly* do what you think it does:
```bash
>>> "%s.%s" % float_split_str(15.00, 2)
'15.00'
>>> "%d.%d" % float_split(15.00, 2)
'15.0'
```
Hopefully the casting to float does the job for you.